packages/ember-data/lib/system/record-array-manager.js

```diff
@@ -160,6 +171,17 @@ export default Ember.Object.extend({
   },
 
   /**
+    Get the `DS.RecordArray` for a type.
+
+    @method liveRecordArrayFor
+    @param {Class} typeClass
+    @return {DS.RecordArray}
+  */
+  liveRecordArrayFor: function(typeClass) {
+    return this.liveRecordArrays.get(typeClass);
+  },
+
+  /**
     Create a `DS.RecordArray` for a type and register it for updates.
```

Comment:
This comment needs to be updated since the *register it for updates* is no more valid ...